misc/snapshotter/snapshotter.sh

```diff
@@ -141,14 +141,14 @@ EOF
                 "${CONTAINER_RUNTIME_CONFIG}".bak
     else
         sed -i '/\[plugins\..*\.containerd\]/a\disable_snapshot_annotations = false' \
-			"${CONTAINER_RUNTIME_CONFIG}".bak
+        "${CONTAINER_RUNTIME_CONFIG}".bak
```

Comment:
Can you make this one and the change on line 151 to follow what's in the line 148?

Reply:
done